packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -257,7 +257,7 @@ export default class SelectBox extends PureComponent {
 
         if (
             displaySearchBox && (
-                !value ||
+                (!value && value !== 0) ||
```

Comment:
i cant tell what `value` is excactly supposed to be when its not selected since it just says `value: PropTypes.any` but i assume its `null`. maybe we should do no type conversion at all and just check if its not null?

as far as i can see `value` can only be a string or an int when it was set (since it was a php array key in the yaml config) 
so it must be null or undefined when nothing is selected right?

Reply:
Adjusted the PR a bit, maybe it is more general for you :)